src/pages/settings/AboutPage/ShareLogPage.tsx

```diff
@@ -0,0 +1,13 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import type {SettingsNavigatorParamList} from '@libs/Navigation/types';
+import type SCREENS from '@src/SCREENS';
+import ShareLogList from './ShareLogList';
+
+type ShareLogPageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.SETTINGS.SHARE_LOG>;
+
+function ShareLogPage({route}: ShareLogPageProps) {
+    return <ShareLogList logSource={route.params.source} />;
```

Comment:
While reviewing another PR, I found out that there will be crash if we open `ShareLogPage` without `source` query param. So I think we should add optional chaining here or wrap `ShareLogPage` with FullPageNotFound view where we don't have `source` query param.
 

https://github.com/Expensify/App/assets/16502320/5847e10d-9780-44fa-89d9-7738981bb5ff